doc/Usage.adoc

```diff
@@ -0,0 +1,272 @@
+= Usage
+
+:toc:
+:toc-placement!:
+
+The following sections explain how you can use the Landrush DNS server and customize it as per your requirements, once you have link:README.adoc[installed] it.
+
+'''
+toc::[]
+'''
+
```

Comment:
We already have links in the TOC taking the user to the relevant section, another clickable table with the same links right below it, might seem too much. Instead we could change the title of the doc to read: 'Usage/Configuration Options' and the text above the TOC could be customized as below, to ensure that the user understands that the TOC is also the list of configuration options

This guide explains how you can use the Landrush DNS server and customize it as per your requirements, once you have link:README.adoc[installed] it. The following sections list the possible configuration options provided by Landrush. 
wdyt?